.github/workflows/weekly-summary.yml

```
@@ -0,0 +1,59 @@
+          
+          # Validate required environment variables
+          if [ -z "$SLACK_WEBHOOK_URL" ] && ([ -z "$SLACK_BOT_TOKEN" ] || [ -z "$SLACK_CHANNEL" ]); then
+            echo "Error: Either SLACK_WEBHOOK_URL or (SLACK_BOT_TOKEN + SLACK_CHANNEL) must be set"
+            echo "For threading support, use: SLACK_BOT_TOKEN and SLACK_CHANNEL"
+            echo "For simple messaging, use: SLACK_WEBHOOK_URL"
+            exit 1
+          fi
+          
+          # Simplified: get commits from last 7 days
+          COMMITS=$(git log --since="7 days ago" --pretty=format:"%an|%s")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)